_transcripts/015-transcription.md

```diff
@@ -0,0 +1,50 @@
+
+This Week in Rust is usually published on Tuesdays and emailed out on Wednesdays. I will most likely be recording this podcast on Monday night so a link to it can be included in the newsletter. So if you want a chance for your story to be highlighted, make sure to get your pull requests in early each week. I am linking to the This Week in Rust GitHub repo in the show notes.
+
+Let’s dive into this week’s highlights.
+
+News & Blog Post
+
+Rust turned 5 this week! It’s been five wonderful years of pursuing faster, more reliable, and safer software. Check out the core team’s blog post looking back on how the language has changed and grown in the open and through the work of our diverse community of individuals. The title of the article is “Five Years of Rust” and it’s the very first item on the News and Blog Post list.
+
+The next highlighted article was actually posted last year, but brought to our attention this week. It’s called “The case for using Rust for Automotive Software” by Sojan James. It is a fascinating article that uses the 2019 fire in Notre Dame Cathedral to illustrate some of the differences between creating automotive software with C++ and creating it with Rust. Definitely check it out!
+
+One of the Rust teams I am a member of is the Infrastructure team. We have been working on migrating to using GitHub actions as our Continuous Delivery and Integration system. Mateus Costa published an article highlighting how he creates Rust releases for single and multiple targets with GitHub actions. It’s a great reference both for getting started with building Rust software through GitHub actions as well as a great introduction to GitHub actions itself.
+
+Something that has encouraged me as much of the world remains in lockdown is seeing how many Rust meetups are still being held online, it is awesome that we can still connect with each other virtually. Many of these meetups are also posting their recordings on Youtube and it’s a great way to see content from around the world. This week’s newsletter features a video of the Rust and C++ Cardiff Virtual meetup in the UK. It features four talks on Rust and C++ as well as some lightning talks. If you’ve been missing your local Rust meetup, check out what’s available online, there’s a lot.
+Finally, I’ve also personally really been enjoying Rust developers streaming their development through YouTube, Twitch, and other services. If you are a newer Rust developer or an advanced developer who just wants to brush up on the fundamentals, you’ll like the called “Jonathan teaches Jason Rust.” You can follow right along as Jonathan Turner - a prominent Rust community member -  takes Jason Turner - A C++ expert - through the basics of Rust development.
```

Comment:
These dashes should probably be em-dash characters.

You can safely paste the unicode "—" character directly into markdown source.  That's what I've been doing, anyway.